app/models/user.rb

```diff
@@ -49,6 +49,8 @@ class User < ApplicationRecord
          :recoverable, :trackable, :validatable
   include DeviseTokenAuth::Concerns::User
 
+  has_many :preferences
```

Comment:
Just some info about [has_many dependent options](https://guides.rubyonrails.org/association_basics.html#options-for-belongs-to-dependent) for adding more consistency to models and associations. For example, when destroying father (user) when belong_to is not optional.